script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -6,82 +6,85 @@ I stoop down in front of Arcueid.
 [sha:fa1e5311dfe889bb0a8186830ca0c5ba26f79877]{
 -- Page 10022, Offset 10422.
 -- 膝と両手を地面につく。
-I put my hands and knees on the earth.
+I put my hands and knees on the ground.
 }
 [sha:91831c54c6ca49ffaaf49a813f9459bc4adbab9d]{
 -- Page 659, Offset 10423.
 -- 地中を走る見えざる<腕|かいな>。
-An invisible arm runs through the ground.
+An invisible arm runs through the soil.
 }
 [sha:b9820fda68b08930686fce2624234b3bc513c5cb]{
 -- Page 659, Offset 10424.
 -- アルクェイドは先ほどと同じ<礫|つぶて>を―――いや、その何倍もの規模で、細かな瓦礫ごと、<こ|・><の|・><地|・><形|・><を|・><変|・><動|・><さ|・><せ|・><た|・>。
-Arcueid transforms the terrain into fine rubble like before―――
-no, the scale this time is %{i}much%{/i} greater.
+Arcueid starts to scoop out a bolder as she did before――
+no, it's of far greater scope this time――
+she scoops out the very ground, rubble included, changing the terrain in the process.
 }
 [sha:7a8ba45340de7d451b6e194b8e00a021e27459df]{
 -- Page 660, Offset 10425.
 -- それは爆発を思わせる、大地そのものの隆起だった。
-The earth itself erupted. You'd be forgiven for thinking a bomb had gone off.
+The earth itself erupts. You'd be forgiven for thinking a bomb had gone off.
 }
 [sha:31a2d2135a66b7f81bd20088656aea4a539bd2cf]{
 -- Page 660, Offset 10426.
 -- 木っ端微塵の力技。ヴローヴの冷気が吹雪なら、アルクェイドの所行は大地の津波だ。
-The power of an overwhelming number of fragments.
+//木っ端微塵の力技 is poetic, and shouldn't be taken too literally, as with most of the coming lines.
+It is a feat of strength that renders all before it to dust.
  If Vlov's cold air is like a blizzard,
  then what Arcueid did is like an earth tsunami.
 }
 [sha:afef4e00d09328d323a22a176d37427de0f746d6]{
 -- Page 661, Offset 10427.
 -- 飛翔する氷の槍など、この壁の如き土砂の前では飛沫にすぎない。
-A flying ice spear is nothing more than a droplet of water before this wall of earth and sand.
+Vlov's paltry soaring spears of ice may as well be as droplets of rain before this wall of earth and sand.
 }
 [sha:c73702798442718915e7264a11a3280ca85d4e3a]{
 -- Page 661, Offset 10428.
 -- 高さ50メートル、7000トンに相当する質量が、ヴローヴを飲み込まんと押し寄せる。
 The mass, 50 meters tall and weighing 7000 tons,
- bears down on Vlov as if to swallow him whole.
+ bears down on Vlov as to swallow him whole.
 }
 [sha:9dd9d12d247e76a4030a2d9b9b1d05a8285ea39c]{
 -- Page 662, Offset 10429.
 -- されど、地を返す天変地異に<騎|ヤ><士|ツ>は動じず。
 And yet,
- even in the face of this natural disaster that has turned the earth's surface inside out,
- the knight doesn't budge.
+ even in the face of this earth-shattering cataclysm,
+ does the knight not budge.
```

Comment:
this word order doesn't really work in this structure. couldn't tell ya why but just "the knight does not budge."